### ISSUE TITLE
The add-on store gives no indication when there is an update to an add-on which is not compatible with this version of NVDA

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#18493]
### Is your feature request related to a problem? Please describe.

Many users will wait before updating to .1 releases of NVDA until add-ons they require are updated.  If an add-on gets an update and is compatible with both the old and new versions of NVDA (2024.4.2 and 2025.1 for instance), a user on the old version (2024.4.2) will be notified and prompted to update and can read the documentation noting that it will work on the new version.

If an add-on is updated, but the new version of the add-on ONLY works on the newer version of NVDA (say 2025.1) then someone on an older version (2024.4.2) will not have any indication that the add-on update they are waiting on (before updating NVDA) has been released.

### Describe the solution you'd like

Ideally for this to be presented to the user as an update but with an indication that it requires a newer version of NVDA to work.

Even more ideally, a user on NVDA 2024.4.2 should have a way of quickly sorting / checking their add-ons and being able to see that say 20 of their add-ons are compatible with NVDA 2025.1, 5 have updates for 2025.1 but which won't work on the older version and 3 have not yet been updated.

### Describe alternatives you've considered

Currently, you can create a portable copy of NVDA, copy your configuration (and thus add-ons) across to it, and update it to 2025.1 in order to see which add-ons are incompatible and also test out the new features of NVDA 2025.1.

It would be preferable for users to be able to see this from within their existing copy of NVDA and without needing to update, particularly as if they inadvertently update their main copy of NVDA, realise their add-ons don't work and then downgrade again, it may (likely) corrupt their configuration.

**Comment by gerald-hartig** [comment]
This is very challenging UX for users in NVDA. We are putting our efforts into the add-on store web front end which should provide that information to users.

**Comment by gerald-hartig** [comment]
Re-opening this issue to better track the underlying user problem. My apologies for the confusion caused by closing this as 'not planned'.

To clarify our strategy: while implementing this feature directly within the NVDA client presents significant UX challenges, we absolutely agree that users need a way to see compatibility information for newer NVDA versions.

Our planned solution for this is a new web-based front-end for the add-on store. We will keep this issue open to track the resolution of the core problem until that web front-end is publicly available and addresses this need.
